wdio/specs/homepage.js

```diff
@@ -138,4 +138,59 @@ describe('Municipality homepage', () => {
     assert(!browser.isVisible('label=כמה סייעות יש בגן ילדים?'));
   });
 
+  it('should open a news element in the same tab, with the hebrew language and residents user type', () => {
+    browser.url('/municipality-1/node/1?user_type=residents&language=he');
```

Comment:
Looks good, but seems like a lot of code duplication. Maybe worth having a help function instead?